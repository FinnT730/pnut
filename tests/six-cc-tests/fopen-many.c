int main() {
  int f;
  char c;
  int i = 0;
  while (i < 100) {
    f = fopen("tests/six-cc-tests/fgetc.c", 0);
    while ((c = fgetc(f)) != -1) {
      putchar(c);
    }
    putchar('\n');
    fclose(f);
    i = i + 1;
  }
}
