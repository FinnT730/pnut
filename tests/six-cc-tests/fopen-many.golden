int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
int main() {
  FILE_ptr f;
  char c;
  f = fopen("six-cc-tests/fgetc.c", 0);
  while ((c = fgetc(f)) != EOF) {
    putchar(c);
  }
  fclose(f);
}
